pkg/auth/authorisation/casbin/Adapter.go

```diff
@@ -119,24 +205,43 @@ func AddPolicy(policies []Policy) []Policy {
 
 func LoadPolicy() {
 	defer handlePanic()
-	err := enforcerImplRef.ReloadPolicy()
+	isCasbinV2, err := enforcerImplRef.ReloadPolicy()
 	if err != nil {
 		fmt.Println("error in reloading policies", err)
 	} else {
-		fmt.Println("policy reloaded successfully")
+		if isCasbinV2 {
+			fmt.Println("V2 policy reloaded successfully")
```

Comment:
let v1 and v2 ReloadPolicy() method log their identity instead logging here

Reply:
Done.